docs/en/contributing/working.md

```diff
@@ -34,7 +36,7 @@ Typical interim commits may look like the following:
 887815f remove toggle underline on hover
 ```
 
-Before submitting a pull request, [rebase](https://www.atlassian.com/git/tutorials/merging-vs-rebasing) your branch from master. Do not use ``git merge``, the *merge* button, or the *update* button provided by Github."
+Before submitting a pull request, [rebase](https://www.atlassian.com/git/tutorials/merging-vs-rebasing) your branch from master. Do not use `git merge`, the _merge_ button, or the _update_ button provided by Github."
```

Comment:
Is the `"` at the end a typo?

Reply:
Fixed. Also fixed the _x_ syntax italics in favor of the *x* syntax.